taf/updater/updater_pipeline.py

```diff
@@ -1215,6 +1216,7 @@ def set_target_repositories_data(self):
 
                 targets_data[repo_name]["commits"] = branch_data
 
+            ensure_pre_push_hook(self.state.users_auth_repo.path)
```

Comment:
I think that this should be a separate step. Because the updater is confusing enough as it is, I'd like all of the steps to be descriptive and as small as possible. So let's add `ensure_pre_push_hook` to the `steps` array. I think that that should be the last step, so that the message that the hook was added is visible without scrolling.